src/chttpd_db.erl

```diff
@@ -93,7 +93,7 @@ handle_changes_req1(#httpd{}=Req, Db) ->
             couch_stats:decrement_counter([couchdb, httpd, clients_requesting_changes])
         end;
     _ ->
-        Msg = <<"Supported `feed` types: normal, continuous, longpoll, eventsource">>,
+        Msg = <<"Supported `feed` types: normal, continuous/stream, longpoll, eventsource">>,
```

Comment:
One thought just crossed my mind: don't you think, that `continuous/stream` could be misread by a user and they'll try to send a request with `?feed=continuous/stream` parameter?